addons/mail/static/src/messaging/widget/form_renderer/form_renderer.scss

```diff
@@ -0,0 +1,26 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_FormRenderer_chatterContainer {
+    display: flex;
+    flex: 1 1 auto;
+    margin: 0 auto;
+    max-width: 1140px;
+    padding: map-get($spacers, 3) map-get($spacers, 3) map-get($spacers, 5);
+    width: 100%;
+}
+
+/*
+    Specificity needed to override generic style made in
+    web_enterprise/static/src/scss/form_view.scss
+*/
+.o_form_view .o_form_sheet_bg .o_FormRenderer_chatterContainer {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+// FIX to hide chatter in dialogs when they are opened from an action returned by python code
+.modal .modal-dialog .o_form_view .o_FormRenderer_chatterContainer {
+    display: none;
+}
```

Comment:
Did we go far enough yet in the refactoring to be able to relatively easily not instantiate the chatter in that case instead of having to hide it?
Out of scope of current PR if not really easy.

Reply:
As explained in the comment, not in case action if returned from python code, in all other cases chatter is not instantiated

Reply:
Which comment is this referring to? I think I missed it.

Reply:
I meant the comment in the code
`// FIX to hide chatter in dialogs when they are opened from an action returned by python code`